src/tools/dartpad/troubleshoot.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Troubleshooting DartPad
+description: Common problems with using DartPad
+---
+
+This page describes solutions to problems that might occur when
+you're trying to use DartPad, whether at [dartpad.dev]({{ site.dartpad }})
+or in a page that has embedded DartPads.
+
+## Embedded DartPads on flutter.dev are slow in Safari
+
+Try using Google Chrome or any other non-Safari
```

Comment:
FWIW, I've taken pains not to emphasize Chrome in the messaging that shows up on DartPad. I really don't want folks to think we're trying to push them toward our browser.